src/Command/CreateSnapshotsCommand.php

```diff
@@ -24,9 +25,18 @@
  * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
  *
  * @final since sonata-project/page-bundle 3.26
+ * NEXT_MAJOR: Remove code related with mode option!
```

Comment:
It's less important since I suppose you'll be the one removing the code on 4.x ; but when we're adding NEXT_MAJOR to remove code we try to add comment on every code to remove

```
NEXT_MAJOR: Remove this line
```
etc.

Reply:
Yeah probably, I'll remove this!

But I'm adding this just for not forget!

Reply:
can you check again please